nova-components/GovernmentManagement/resources/js/tool.js

```diff
@@ -0,0 +1,14 @@
+import StoreModule from "../store/store-module"
+
+Nova.booting((Vue, router, store) => {
+  router.addRoutes([
+    {
+      name: 'government-management',
+      path: '/government-management',
+      component: require('./components/Tool'),
+    },
+  ])
+
+  store.registerModule("nova-store/module", StoreModule)
```

Comment:
module name should be describing it's content (e.g. GovernmentManagement module)